test/integration/008_schema_tests_test/models-v2/models/schema.yml

```diff
@@ -9,6 +9,9 @@ models:
           tests:
             - not_null
             - unique
+            - accepted_values:
```

Comment:
The dbt test suite generally asserts that the number of run tests == the number of expected tests. We had some... issues... a while back where we had a bunch of tests that didn't actually run!

I'll kick off the test suite now, but there might be one more tiny tweak to make in the Python part of the integration tests - I'll follow up when those have finished running.

Reply:
yeah, check out the failure here:
```
self = <test_schema_v2_tests.TestSchemaTests testMethod=test_postgres_schema_tests>

    @use_profile('postgres')
    def test_postgres_schema_tests(self):
        results = self.run_dbt()
        self.assertEqual(len(results), 5)
        test_results = self.run_schema_validations()
        # If the disabled model's tests ran, there would be 19 of these.
>       self.assertEqual(len(test_results), 18)
E       AssertionError: 19 != 18

test/integration/008_schema_tests_test/test_schema_v2_tests.py:35: AssertionError
```

If you just change the `18` in there to a `19`, I think this will be ready to roll :)

Reply:
Ah, fascinating. Magic number updated!